slack_wrapper.gs

```diff
@@ -264,19 +175,57 @@ class SlackEventWrapper {
       var immediateReturnMessage = "Thank you for your message. I\'m a poor bot so please be patient... it should take me up to a few minutes to get back to you...";
```

Comment:
`immediateReturnMessage` can probably be remove both here and from `processFunctionAsync` and replaced directly with `return_message`. The way it is, it looks like it might be used for something more than it is.

While there it might make sense to move the reply url argument inside of the `ProcessFunctionAsync` so the signiture is the same as `processFunctionSync`.

Reply:
done in new commit